documentation/Installing-Clear-Containers-on-ClearLinux.md

```diff
@@ -12,20 +12,20 @@ $ sudo swupd update
 ## Install the Clear Containers bundle
```

Comment:
Hmm, on my screen that is showing as next to the 'install bundle' line - but the comment was aimed at the subject (first) line - the 'Install the Clear Containers runtime on Clear Linux' line.